### PR TITLE
[Update & New] Getting started SELinux

### PR DESCRIPTION
**Pull request by leslitagordita**
Update: CentOS 7 version
New: CentOS 8 version

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 89eb861ea7ec94fc869a8c0beccbf50017a8130d

https://deploy-preview-3212--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/security/selinux/a-beginners-guide-to-selinux-on-centos-7/index.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+# restorecon -v 'logs'
+    {{</ output >}}
+
+- The example was generated using a [Linode running the Apache webserver](/docs/web-servers/lamp/how-to-install-a-lamp-stack-on-centos-7/#apache) with a virtual hosts configuration.
+
+- To allow `/usr/sbin/httpd` write access to the directory logs, as shown by the output, you would issue the suggested commands, `semanage fcontext -a -t httpd_sys_rw_content_t 'logs'` and `restorecon -v 'logs'`.
+
+## SELinux Context
+
+SELinux marks every single object on a machine with a *context*. Every file, user, and process has a context. The context is broken into three parts: *user*, *role*, and *type*. An SELinux policy controls which users can get which roles. Each specific role places a constraint on what type of files that user can access. When a user logs in to a system, a role is assigned to the user. which can be seen in the `ls -Z` example above: the output `unconfined_u` is a user role.
```
Comment (andystevensname) [inline docs/security/selinux/a-beginners-guide-to-selinux-on-centos-7/index.md]: Errant period after "a role is assigned to the user." Probably meant to be a comma.

File: docs/security/selinux/a-beginners-guide-to-selinux-on-centos-8/index.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+# restorecon -v 'logs'
+    {{</ output >}}
+
+- The example was generated using a [Linode running the Apache webserver](/docs/web-servers/lamp/how-to-install-a-lamp-stack-on-centos-7/#apache) with a virtual hosts configuration.
+
+- To allow `/usr/sbin/httpd` write access to the directory logs, as shown by the output, you would issue the suggested commands, `semanage fcontext -a -t httpd_sys_rw_content_t 'logs'` and `restorecon -v 'logs'`.
+
+## SELinux Context
+
+SELinux marks every single object on a machine with a *context*. Every file, user, and process has a context. The context is broken into three parts: *user*, *role*, and *type*. An SELinux policy controls which users can get which roles. Each specific role places a constraint on what type of files that user can access. When a user logs in to a system, a role is assigned to the user. which can be seen in the `ls -Z` example above: the output `unconfined_u` is a user role.
```
Comment (andystevensname) [inline docs/security/selinux/a-beginners-guide-to-selinux-on-centos-8/index.md]: Errant period after "assigned to the user."

File: docs/security/selinux/a-beginners-guide-to-selinux-on-centos-7/index.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+httpd_can_connect_zabbix --> off
+httpd_can_network_connect --> off
+httpd_can_network_connect_cobbler --> off
+httpd_can_network_connect_db --> off
+httpd_can_network_memcache --> off
+httpd_can_network_relay --> off
+httpd_can_sendmail --> off
+    {{</ output >}}
+
+    You can change the value of any variable using the `setsebool` command. If you set the `-P` flag, the setting will persist through reboots. If, for example, you want to allow HTTPD scripts and modules to connect to the network update the corresponding boolean variable
```
Comment (andystevensname) [inline docs/security/selinux/a-beginners-guide-to-selinux-on-centos-7/index.md]: Comma after network

File: docs/security/selinux/a-beginners-guide-to-selinux-on-centos-8/index.md [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+httpd_can_connect_zabbix --> off
+httpd_can_network_connect --> off
+httpd_can_network_connect_cobbler --> off
+httpd_can_network_connect_db --> off
+httpd_can_network_memcache --> off
+httpd_can_network_relay --> off
+httpd_can_sendmail --> off
+    {{</ output >}}
+
+    You can change the value of any variable using the `setsebool` command. If you set the `-P` flag, the setting will persist through reboots. If, for example, you want to allow HTTPD scripts and modules to connect to the network update the corresponding boolean variable
```
Comment (andystevensname) [inline docs/security/selinux/a-beginners-guide-to-selinux-on-centos-8/index.md]: Comma after network

File: docs/security/selinux/a-beginners-guide-to-selinux-on-centos-8/index.md [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+SELinux defaults to denying anything that is not explicitly allowed. SELinux has two global modes, *permissive* and *enforcing*. Permissive mode allows the system to function like a DAC system, while logging every violation to SELinux. The enforcing mode applies a strict denial of access to anything that isn't explicitly allowed. To explicitly allow certain behavior on a machine, you, as the system administrator, have to write policies that allow it. This guide provides a brief and basic introduction to commonly used commands and practices for SELinux system administration.
+
+## Before You Begin
+
+1.  Update your system:
+
+        yum update
+
+    {{< note >}}
+The Linode kernel does not support SELinux by default. However, all new Linodes running CentOS 7 use the distribution provided kernel, which has **SELinux enabled by default**.
```
Comment (andystevensname) [inline docs/security/selinux/a-beginners-guide-to-selinux-on-centos-8/index.md]: Change CentOS 7 to 8

File: docs/security/selinux/a-beginners-guide-to-selinux-on-centos-8/index.md [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+
+## Install Supporting SELinux Packages
+
+In this section, you will install various SELinux packages that will help you when creating, managing, and analyzing SELinux policies.
+
+1. Verify which SELinux packages are installed on your system:
+
+        rpm -aq | grep selinux
+
+    A newly deployed CentOS 7 Linode should have the following packages installed:
```
Comment (andystevensname) [inline docs/security/selinux/a-beginners-guide-to-selinux-on-centos-8/index.md]: CentOS 8

File: docs/security/selinux/a-beginners-guide-to-selinux-on-centos-8/index.md [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+    - `setools` provides command line tools for working with SELinux policies. Some of these tools include, `sediff` which you can use to view differences between policies, `seinfo` a tool to view information about the components that make up SELinux policies, and `sesearch` used to search through your SELinux policies. You can issue the `--help` option after any of the listed tools in order to view more information about each one.
+    - `setroubleshoot` suite of tools help you determine why a script or file may be blocked by SELinux.
+
+    Optionally, install `setroubleshoot-server` and `mctrans`. The `setroubleshoot-server` allows, among many other things, for email notifications to be sent from the server to notify you of any policy violations. The `mctrans` daemon translates SELinux's output to human readable text.
+
+## SELinux States and Modes
+
+### SELinux States
+
+When SELinux is installed on your system, it can be either *enabled* or *disabled*. By default, the CentOS 7 image provided by Linode will have SELinux in an enabled state.
```
Comment (andystevensname) [inline docs/security/selinux/a-beginners-guide-to-selinux-on-centos-8/index.md]: CentOS 8

**Pull request merged**